alternator/executor.cc

```diff
@@ -423,7 +423,7 @@ sstring executor::table_name(const schema& s) {
     return s.cf_name();
 }
 
-void executor::fill_table_description(schema_ptr schema, table_status tbl_status, rjson::value& table_description) const
+void executor::fill_table_description(schema_ptr schema, table_status tbl_status, service::storage_proxy const& proxy, rjson::value& table_description)
```

Comment:
This can now be a static (file-local) function and doesn't need to be a static (class-local) "executor" method.

Reply:
Done. Also moved table_status enum to executor.cc